install/updateVersion.sh

```diff
@@ -134,30 +151,42 @@ export MIXER_TAG="${MIXER_TAG}"
 export ISTIOCTL_URL="${ISTIOCTL_URL}"
 export PILOT_HUB="${PILOT_HUB}"
 export PILOT_TAG="${PILOT_TAG}"
+export ISTIO_NAMESPACE=${ISTIO_NAMESPACE}
 EOF
 }
 
 function update_istio_install() {
   pushd $TEMP_DIR/templates
```

Comment:
oh I missed that initially, everything is copied in /tmp/templates and modified there
it probably should be a unique temp dir instead but I guess that's not a new problem created by this pr

hopefully we can agree this script can be improved and replaced by something less convoluted (not in this PR clearly, but for 0.3)

Reply:
Yes, sorry about the existence of this script :) The problem with these bash scripts is that they tend to grow beyond their original purpose. We should try to address the core problems (auth vs no auth) and obviate the need to customize installations.

Reply:
+1, it's a pain to write bash